BDArmory/Radar/ModuleRadar.cs

```diff
@@ -100,6 +100,8 @@ public class ModuleRadar : PartModule
         [KSPField]
         public float radarGroundClutterFactor = 0.25f; //Factor defining how effective the radar is for look-down, compensating for ground clutter (0=ineffective, 1=fully effective)
                                                        //default to 0.25, so all cross sections of landed/splashed/submerged vessels are reduced to 1/4th, as these vessel usually a quite large
+        [KSPField]
+        public int sonarType = 0; //0 = Radar; 1 == Active Sonar; 2 == Passive Sonar
```

Comment:
Making an enum for this (similar to `rwrType`) would probably make other parts of the code more readable.
Though it looks like `rwrThreatType` is used in a lot of places where `rwrType` ought to be being used instead.